test_inventories.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from unittest.mock import MagicMock
-from inventory import Fetcher
-from inventory import InventoryBuilder
+from maas import Fetcher
```

Comment:
just reminded that for portability we should be using ./maas instead of maas. the name is to generic and there can be easily other packages named maas in the python path